### ISSUE TITLE
navBarCustomView height

### ISSUE THREAD

**Issue by Kosai106** [wix/react-native-navigation#1844]
### Issue Description

Hi, I'm trying to use a custom component for the navBar, which I can do successfully using the navBarCustomView. However, when setting a different height for the view using `navBarHeight`, the component within the view isn't adjusting it's size regardless of flex or height/width being set.

Basically I want the light purple to fill as much of the dark purple area as possible.
Screenshot: https://cdn.discordapp.com/attachments/339371162855276544/356561261691011072/21616786_10214856714151033_1312785356_o.png

### Steps to Reproduce / Code Snippets / Screenshots

index.js
```javascript
import { Navigation } from 'react-native-navigation'

import { iconsMap, iconsLoaded, customIcon } from './utils/icons'
import { registerScreens } from './screens'

registerScreens()

iconsLoaded.then(() => {
    startApp()
})

function startApp() {
  Navigation.startSingleScreenApp({
    drawer: {
      left: {
        screen: 'example.Drawer'
      },
      style: {
        leftDrawerWidth: 85,
      }
        },
    screen: {
      title: 'Banking',
      screen: 'example.Container',
      navigatorStyle: {
        navBarCustomView: 'example.CustomHeader'
      },
      topTabs: [
        {
          screenId: 'example.ScreenOne',
          title: 'Recent'
        },
        {
          screenId: 'example.ScreenTwo',
          title: 'This month'
        },
        {
          screenId: 'example.ScreenThree',
          title: '3 months'
        }
      ],
      leftButtons: [{
        id: 'sideMenu',
        title: 'Menu',
        icon: iconsMap['menu'],
      }]
    },
    appStyle: {
      orientation: 'portrait',
      navBarTextColor: '#FFFFFF',
      navBarButtonColor: '#FFFFFF',
      navBarBackgroundColor: '#221B36',
      navBarHeight: 250,
      topTabTextColor: 'rgba(255,255,255,0.5)',
      selectedTopTabTextColor: 'rgba(255,255,255,1.0)',
      selectedTopTabIndicatorHeight: 8,
      selectedTopTabIndicatorColor: '#39B5E8',
      topTabsScrollable: false,
      statusBarColor: '#221B36',
      topBarElevationShadowEnabled: false
    }
  })
}
```

customHeader.js
```javascript
import React, { Component } from 'react'
import { StyleSheet, Text, View, StatusBar } from 'react-native'

export default class CustomHeader extends Component {
  render() {
    return (
      <View style={styles.container}>
        <StatusBar barStyle={'light-content'} translucent={false} backgroundColor='#221B36' />
        <Text style={styles.largeFont}>
        Current Account
        </Text>
        <Text style={styles.mutedFont}>
            Private &middot; 3872 4839226841
        </Text>
        <Text style={styles.largeFont}>
            Your Balance
        </Text>
        <Text style={styles.mutedFont}>
            $ 5.827,32
        </Text>
      </View>
    )
  }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: 'center'
    },
    largeFont: {
        fontSize: 16,
        color: '#FFFFFF'
    },
    mutedFont: {
        fontSize: 12,
        color: 'rgba(255,255,255,0.75)'
    },
})
```

---
### Environment
* React Native Navigation version: 1.1.213
* React Native version: 0.48.2
* Platform(s) (iOS, Android, or both?): Android
* Device info (Simulator/Device? OS version? Debug/Release?): Device, Android 7.1.1, Debug

**Comment by guyca** [comment]
Hey @Kosai106 
I'll get to this tomorrow

**Comment by Kosai106** [comment]
@guyca I see you've closed the issue. May I ask what were your findings? I'm still stuck with this issue.

**Comment by guyca** [comment]
The react view wasn't using the navBarHeight property, instead it was given the default toolbar height. This fix is published in v1.1.222
